src/book/01-team/02-community/01-sponsors.mdx

```diff
@@ -5,3 +5,15 @@ title: Sponsors
 description: The companies and organisations that fund NUbots research.
 slug: /team/sponsors
 ---
+
+![](.../.../.../images/UoN.png)
+
+![](.../.../.../images/4Tel.png)
+
+![](.../.../.../images/Tribotix.png)
+
+![](.../.../.../images/Elby.png)
+
+![](.../.../.../images/EMONA.png)
```

Comment:
We could just normalise all file names to lowercase:

```md
![](.../.../.../images/emona.png)
```

Reply:
Done :+1: